docker/StateFile.yml

```diff
@@ -0,0 +1,141 @@
+Kind: Deployment
+ApiVersion: v0.8
+
+Namespace: global
+
+Cargoes:
+  - Name: db
+    Container:
+      Tty: true
+      Image: postgres:12
+      Env:
+        - POSTGRES_PASSWORD=docker
+        - PGPASSWORD=docker
+
+  - Name: dbmate
+    Container:
+      Tty: true
+      Image: docker-dbmate:latest
+      HostConfig:
+        AutoRemove: true
+        Dns:
+          - ${{ Namespaces.global.Gateway }}
+          - 1.1.1.1
+        Binds:
+          - ../dbmate/db:/app/db
+      Env:
+        - DATABASE_URL=postgres://postgres:docker@db.zoom.rs:5432/actix-api-db?sslmode=disable
+
+  - Name: ui
+    Container:
+      Tty: true
+      Image: docker-yew-ui:latest
+      WorkingDir: /app/yew-ui
+      Cmd:
+        - bash
+        - -c
+        - trunk serve --address 0.0.0.0 --port 8081
+      HostConfig:
+        Dns:
+          - ${{ Namespaces.global.Gateway }}
+          - 1.1.1.1
+        Binds:
+          - ../:/app
+          - ${{ Envs.HOME }}/.cargo/registry:/usr/local/cargo/registry
+      Env:
+      - ACTIX_HOST=api.zoom.rs
+      - ACTIX_PORT=8080
+      - ACTIX_UI_BACKEND_URL=ws://api.zoom.rs
+      - TRUNK_SERVE_PORT=8081
+      - ENABLE_OAUTH=false
+      - LOGIN_URL=http://api.zoom.rs/login
+      - RUSTFLAGS=--cfg=web_sys_unstable_apis
+      - RUST_BACKTRACE=1
```

Comment:
Does this need to be indented once?

Reply:
Both are valid, you can do a `nanocl cargo inspect {cargo_name}` to verify if it's correctly set